### PR TITLE
New blog post about a code review exercise

### PR DESCRIPTION
**Pull request by dpshelio**
This blog post relates the experiences of a couple of code review exercises I've done at my work place.

@gvwilson  I hope you are not too mad that I've done it 8 hours after the deadline.

### CODE REVIEW

File: blog/2015/10/code-review-habit.html [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+<!-- start excerpt -->
+<p>
+  More than a year ago, Marian Petre and Greg Wilson wrote an article
+about <a href="http://arxiv.org/abs/1407.5648">reviewing code</a>, but
+I was not aware of it till this week when Greg talked about their
+findings in a workshop in London
+(though <a href="{{page.root}}/blog/2014/01/code-review-round-2.html">it
+had been mentioned here before</a>).  I have been reviewing code for
+years, however I have not realised that I was doing so till GitHub
+made it easy. When I read code from others feels the same way that
```
Comment (jiffyclub) [inline blog/2015/10/code-review-habit.html]: I'd change this to "Reading code from others feels the same as proofreading a Spanish text, ..."

File: blog/2015/10/code-review-habit.html [hunk 2]
```diff
@@ -0,0 +1,126 @@
...
+when I am proofreading a Spanish text, where you have to pay attention
+to the orthography, the grammar and that it says what it's meant to
```
Comment (jiffyclub) [inline blog/2015/10/code-review-habit.html]: "that" -> "whether"

File: blog/2015/10/code-review-habit.html [hunk 3]
```diff
@@ -0,0 +1,126 @@
...
+say (or it does what it's meant to do).
+  </p>
+<!-- end excerpt -->
+
+Since I started my PhD I've organised <emph>programming clubs</emph> in the
+institutes where I've been as a way of sharing knowledge between the
+colleagues. But in these last 10 years I've just done a "review
```
Comment (jiffyclub) [inline blog/2015/10/code-review-habit.html]: review code -> code review ?

**Comment by dpshelio** [comment]
Thanks @jiffyclub for your comments. Please, any other comments/suggestions are more than welcome. First time I write for the blog :)

File: blog/2015/10/code-review-habit.html [hunk 4]
```diff
@@ -0,0 +1,126 @@
...
+<p>
+  More than a year ago, Marian Petre and Greg Wilson wrote an article
+about <a href="http://arxiv.org/abs/1407.5648">reviewing code</a>, but
+I was not aware of it till this week when Greg talked about their
+findings in a workshop in London
+(though <a href="{{page.root}}/blog/2014/01/code-review-round-2.html">it
+had been mentioned here before</a>).  I have been reviewing code for
+years, however I have not realised that I was doing so till GitHub
+made it easy. Reading code from others feels the same as proofreading
+a Spanish text, where you have to pay attention to the orthography,
```
Comment (rgaiacs) [inline blog/2015/10/code-review-habit.html]: Just one suggestion: "a Spanish text" -> "a Spanish text (or any other text)".

File: blog/2015/10/code-review-habit.html [hunk 5]
```diff
@@ -0,0 +1,126 @@
...
+the grammar and whether it says what it's meant to say (or it does what
+it's meant to do).
+  </p>
+<!-- end excerpt -->
+
+Since I started my PhD I've organised <emph>programming clubs</emph> in the
```
Comment (rgaiacs) [inline blog/2015/10/code-review-habit.html]: Missing`<p>` for this block.

File: blog/2015/10/code-review-habit.html [hunk 6]
```diff
@@ -0,0 +1,126 @@
...
+for our future-self to understand what we wrote.
+</p>
+<p>
+  In summary I don't think I can add much more to what is in Marian and
+Greg's paper but to emphasise the need to do such activity in our
+research groups, to raise its importance as when we review papers as
+these codes are the key for science progress.
+</p>
+
+I think it's worth mention different justifications I've heard during
```
Comment (rgaiacs) [inline blog/2015/10/code-review-habit.html]: Missing `<p>` at this block.

File: blog/2015/10/code-review-habit.html [hunk 7]
```diff
@@ -0,0 +1,126 @@
...
+the years to don't share code like:
+<ul>
+ <li> it's not well commented and you won't understand it;
+ <li> I will show the code when the paper is published;
+ <li> The code has some drawn-backs and if I leave it there to people
+ to use they will publish wrong things without understanding it.;
+ <li> I want people to contact me to run it, so I can be in their publications.;
+</ul>
+
+I believe that if more researchers get used to review code, many of
```
Comment (rgaiacs) [inline blog/2015/10/code-review-habit.html]: Again, missing `<p>` at this block.

**Comment by rgaiacs** [comment]
@dpshelio Very good first blog post. :sparkles:

**Comment by dpshelio** [comment]
Thanks @rgaiacs!!!

**Pull request merged**